modules/client/src/client-common/constants.ts

```diff
@@ -138,3 +143,24 @@ export const LIVE_CONTRACTS: { [K in SupportedNetwork]: NetworkDeployment } = {
     ensRegistry: activeContractsList.polygon.ENSRegistry,
   },
 };
+
+export const FAILING_PROPOSAL_AVAILABLE_FUNCTION_SIGNATURES = [
+  Multisig__factory.createInterface().getFunction("addAddresses")
```

Comment:
Refactor the repeated `Multisig__factory.createInterface()` into a local constant